src/aind_metadata_mapper/core.py

```diff
@@ -236,10 +236,10 @@ def from_args(cls, args: list):
         parser = argparse.ArgumentParser()
         parser.add_argument(
             "-i",
-            "--input-empty_source",
+            "--input-source",
```

Comment:
Open a new ticket if source code needs to be modified as this pr is just for updating docs

Reply:
This was changed during an auto refactor. It isn't in the latest commit